flutter_src/lib/logic/state/bloc_validator.dart

```diff
@@ -31,12 +48,14 @@ class CustomBlocValidator {
     List<String> emails = string.split(',');
 
     for (int i = 0; i < emails.length; i++) {
-      bool emailValid = RegExp(r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+").hasMatch(emails[i]);
+      bool emailValid = RegExp(
```

Comment:
nit: typically you want to trim whitespaces from emails[i] here in case the user added spaces after the comma.